hack/force-cert-retation.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+set -xe
+
+KUBECTL=${KUBECTL:-kubectl}
+POD_NAMESPACE=${POD_NAMESPACE:-kubemacpool-system}
+POD_APP_LABEL=${POD_APP_LABEL:-kubemacpool-leader=true}
+WEBHOOK_TYPE=${WEBHOOK_TYPE:-mutatingWebhookConfiguration}
+WEBHOOK_NAME=${WEBHOOK_NAME:-kubemacpool-mutator}
+TEMP_FOLDER=${TEMP_FOLDER:-_tmp}
+
+WEBHOOK_CONFIG_YAML_DIR=$(pwd)/${TEMP_FOLDER}/
+WEBHOOK_YAML_FILE=${WEBHOOK_CONFIG_YAML_DIR}/${WEBHOOK_TYPE}_${WEBHOOK_NAME}.yaml
+
+mkdir -p ${WEBHOOK_CONFIG_YAML_DIR}
+
+# get webhook config in yaml format
+${KUBECTL} get ${WEBHOOK_TYPE} -n ${POD_NAMESPACE} ${WEBHOOK_NAME} -oyaml > ${WEBHOOK_YAML_FILE}
```

Comment:
You can remove caBundle using `kubectl` with JSON patch.

Reply:
Examples of using it:

https://kubernetes.io/docs/reference/kubectl/cheatsheet/#patching-resources

Reply:
In order to do that, I need to know the specific location of the caBundle, so I either need to add the webhooks names as env var or try to iterate on them using shyaml (not sure they support. sed was simpler.. 
What do you think? is there another way?

Reply:
well let me try first before I say it hard :)

Reply:
I think you can use ".." to do nested CABundle patch at least jsonpath use that.

Reply:
But we can have multiple webhooks, so `..` would give us only the first one, no?